File: Linkpoint/src/main/java/com/linkpoint/slproto/modules/rlv/commands/RLVCmdSendChannel.java
package com.linkpoint.slproto.modules.rlv.commands;

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType;

public class RLVCmdSendChannel extends RLVCmdGenericRestriction {
    public RLVCmdSendChannel() {
        super(RLVRestrictionType.sendchannel, true);
    }
}

File: Linkpoint/src/main/kotlin/com/linkpoint/slproto/modules/rlv/commands/RLVCmdSendChannel.kt
package com.linkpoint.slproto.modules.rlv.commands

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType

class RLVCmdSendChannel : RLVCmdGenericRestriction() {
    public RLVCmdSendChannel() {
        super(RLVRestrictionType.sendchannel, true)
    }
}
